docs/site/content/docs/assets/package-installation.md

```diff
@@ -1,15 +1,29 @@
 ## Installing a Package
 
-This section walks you through installing a package (cert-manager) in your cluster. For
-detailed instruction on package management, see [Work with Packages](../package-management).
+This section walks you through installing the [cert-manager](https://cert-manager.io/docs/) package in your cluster as an example of package installation. For detailed instruction on package management, see [Work with Packages](../package-management).
```

Comment:
I think this speaks to part of the survey respondent's request. Do we want to add a sentence here that says what cert-manager is or does instead of solely linking out?